docs/basics/101-140-filesystem.rst

```diff
@@ -0,0 +1,764 @@
+dropped automatically, but for content in sub-*datasets* to be dropped, the
+``-r/--recursive`` flag has to be included.
+
+By default, DataLad will not drop any content that does not have at least
+one verified remote copy that the content could be retrieved from again.
+It is possible to drop the downloaded image, because thanks to
+:command:`datalad download-url` its original location in the web in known:
+
+.. runrecord:: _examples/DL-101-140-175
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   $ datalad drop flowers.jpg
+
+Currently, the file content is gone, and opening the remaining symlink
+will fail. But the content can be obtained easily again with
+:command:`datalad get`:
+
+.. runrecord:: _examples/DL-101-140-176
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   $ datalad get flowers.jpg
+
+If a file has no verified remote copies, DataLad will only drop its
+content if the ``--nocheck`` option is specified. We will demonstrate
+this by generating a random PDF file:
+
+.. runrecord:: _examples/DL-101-140-177
+   :workdir: dl-101/DataLad-101
+   :language: console
+
+   $ convert xc:none -page Letter a.pdf
+   $ datalad save -m "add empty pdf"
+
+DataLad will safeguard dropping content that it can't retrieve again:
+
+.. runrecord:: _examples/DL-101-140-178
+   :workdir: dl-101/DataLad-101
+   :language: console
+
+   $ datalad drop a.pdf
+
+But with the ``--nocheck`` flag it will work:
+
+.. runrecord:: _examples/DL-101-140-179
+   :workdir: dl-101/DataLad-101
+   :language: console
+
+   $ datalad drop --nocheck a.pdf
+
+Note though that this file content is irreversibly gone now, and
+even going back in time in the history of the dataset will not bring it
+back into existence.
+
+Finally, let's clean up:
+
+.. runrecord:: _examples/DL-101-140-180
+   :workdir: dl-101/DataLad-101
+   :language: console
+
+   $ git reset --hard HEAD~2
+
+Uninstalling or deleting subdatasets
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Depending on the exact aim, two commands are of relevance if anyone wants to
+delete a DataLad subdataset. The softer (and not so much "deleting" version)
+is to uninstall a dataset with the :command:`datalad uninstall` (:manpage:`datalad-uninstall` manual)
+This command can be used to uninstall any number of
+*subdatasets* -- note though that only subdatasets can be uninstalled, the command
+will error if given a sub-*directory*, a file, or a top-level dataset.
+
+.. runrecord:: _examples/DL-101-140-160
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   # Install a subdataset - the content is irrelevant, so why not a cloud :)
+   $ datalad install -d . \
+    --source https://github.com/datalad-datasets/disneyanimation-cloud.git \
+    cloud
+
+To uninstall the dataset, use
+
+.. runrecord:: _examples/DL-101-140-161
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   $ datalad uninstall cloud
+
+Note that the dataset is still known in the dataset, and not completely removed.
```

Comment:
I guess, this might need a sentence or two, i.e. what is deleted in what sense and what is not.
May be just point out, that you can get it again in the exact same state as before via `datalad install` (or `datalad get`), which is not true, if it's "really" removed from the dataset.